ros_ign_interfaces/msg/ContactState.msg

```diff
@@ -0,0 +1,8 @@
+string info                                   # text info on this contact
+string collision1_name                        # name of contact collision1
+string collision2_name                        # name of contact collision2
+geometry_msgs/Wrench[] wrenches               # list of forces/torques
+geometry_msgs/Wrench total_wrench             # sum of forces/torques in every DOF
```

Comment:
I'd also suggest removing the `total_wrench` field, because it doesn't add new information. The receiver can calculate it if they need it.